.travis.yml

```diff
@@ -1,6 +1,17 @@
-language: node_js
-services: mongodb
-sudo: false
-node_js:
-  - "8"
-  - "10"
+language: node_js
+services: mongodb
+sudo: true
+before_install:
+  - sudo apt-get update -q
+  - sudo apt-get install python3-pip python-dev -y
+  - pip3 install -U setuptools
+  - pip3 install -U virtualenvwrapper
+install:
+  - nvm install-latest-npm
+  - npm install
+  - pip3 install -r requirements.txt
+node_js:
+  - "8"
+  - "10"
+python:
+  - "3.5"
```

Comment:
Why are you installing python manually and adding it here? Which one is actually used?

Reply:
Actually, python 3.5 is being used. Should get rid of `python-dev`, as might not be needed for any of our requirements.

Reply:
To verify that python3.5 is being selected, I have added a test like this:
```javascript
it('should find python 3.5', () => {
        const processOutput = spawnSync('python3', ['-c', `import sys; assert sys.version[0:3] == '3.5'`]);
        assert.equal(processOutput.status, 0);
    });
```